Create new token description (#282)
sora-xor/sora-docs: src/pussy.md

---
title: PUSSY | SORA Docs
head:
  - - meta
    - name: description
      content: Learn about the PUSSY token, a dedicated cryptocurrency designed to fund and reward builders within our community
  - - meta
    - name: keywords
      content: reward, mint, Kensetsu, XOR, KXOR
---

# PUSSY

<img src=".gitbook/assets/pussy_image.png" alt="drawing" width="500"/>

## Introduction

> The PUSSY token is a dedicated cryptocurrency in the SORA community, aimed at funding and rewarding builders. It plays a crucial role in the ecosystem by ensuring builders are compensated for their efforts and can convert their earnings into a stable and valuable asset.

Reviewer: ```suggestion
The PUSSY token is a dedicated meme cryptocurrency in the SORA economy aimed at funding and rewarding builders. Its role in the ecosystem is to ensure builders are compensated for their efforts and can convert their earnings into a stable and valuable asset.
```

---
title: PUSSY | SORA Docs
head:
  - - meta
    - name: description

>       content: Learn about PUSSY, reward builders, mint Kensetsu XOR (KXOR)

Reviewer: ```suggestion
      content: Learn about the PUSSY token, a dedicated cryptocurrency designed to fund and reward builders within our community
```

  - - meta
    - name: keywords
      content: reward, mint, Kensetsu, XOR, KXOR
---

> # PUSSY :cat:

Reviewer: A nice cat image would be good

## Introduction

> The PUSSY token is a dedicated cryptocurrency designed to fund and reward builders within our community. Here’s an overview of its purpose and functionality

Reviewer: ```suggestion
The PUSSY token is a dedicated cryptocurrency designed to fund and reward builders within the SORA community. Here’s an overview of its purpose and functionality
```

## Purpose of PUSSY Tokens

> The PUSSY token is created specifically to support builders working on productive outputs and projects within the SORA ecosystem.

Reviewer: ```suggestion
The PUSSY token supports builders working on productive outputs and projects within the SORA ecosystem.
```
